pkg/server/sandbox_run.go

```diff
@@ -86,8 +87,15 @@ func (c *criContainerdService) RunPodSandbox(ctx context.Context, r *runtime.Run
 	glog.V(4).Infof("Sandbox container spec: %+v", spec)
 	// TODO(random-liu): Checkpoint metadata into container labels.
 
+	b,err := sandbox.Metadata.Encode()
+	if err != nil {
+		return nil, fmt.Errorf("failed to convert sandbox metadata: %+v, %v", sandbox.Metadata, err)
+	}
+	labels[sandboxMetadataLabel] = string(b)
```

Comment:
Actually the `Pid`, `NetNS` and `CreatedAt` underlying are not included here.

I'm fine with this for now. After this PR is merged, we could:
1) Get `Pid` from containerd, so that we don't need to checkpoint it ourselves;
2) Use permanent `NetNS` after https://github.com/kubernetes-incubator/cri-containerd/pull/138 is merged, so that we'll have network namespace here;
3) Get `CreatedAt` from containerd, which have been checkpointed by containerd https://github.com/containerd/containerd/blob/master/containers/containers.go#L14.

Please add a TODO for this.